pkg/monitoring/rules/recordingrules/virt.go

```diff
@@ -45,7 +45,7 @@ func virtRecordingRules(namespace string) []operatorrules.RecordingRule {
 			},
 			MetricType: operatormetrics.GaugeType,
 			Expr: intstr.FromString(
-				fmt.Sprintf("sum(up{pod=~'virt-controller-.*', namespace='%s'}) or vector(0)", namespace),
+				fmt.Sprintf("sum(kube_pod_status_phase{pod=~'virt-controller-.*', namespace='%s', phase='Running'}) or vector(0)", namespace),
 			),
```

Comment:
what about scheduled pods shouldn't we count these?

Reply:
it was only considered `up` if the target was running and the metrics scrape was successful, keeping similar behaviour